moveit_ros/hybrid_planning/local_planner/local_planner_component/src/local_planner_component.cpp

```diff
@@ -96,6 +96,12 @@ bool LocalPlannerComponent::initialize()
   planning_scene_monitor_->startWorldGeometryMonitor(config_.collision_object_topic);
   planning_scene_monitor_->startStateMonitor(config_.joint_states_topic);
 
+  // Create a copy of the planning scene
+  planning_scene_monitor_->lockSceneRead();  // LOCK planning scene
```

Comment:
use `LockedPlanningSceneRO` over the `lock/unlock` methods. What happens if you hit an exception in between? :/